Dockerfile

```diff
@@ -73,7 +72,7 @@ ENV JAVA_HOME=/opt/graalvm
 COPY --chown=ksml:0 --from=graal-builder /opt/graal/ /opt/graal/
 
 WORKDIR /home/ksml
-USER 1024
+USER 1024:0
```

Comment:
This is not required as the user is created in root group already.

Reply:
Removed the explicit group id